src/main/java/com/puppycrawl/tools/checkstyle/ConfigurationLoader.java

```diff
@@ -492,7 +471,15 @@ private final class InternalLoader
          */
         private InternalLoader()
                 throws SAXException, ParserConfigurationException {
-            super(createIdToResourceNameMap());
+            super(Map.of(
```

Comment:
Please add comment like `// map from local resources and dtd ids` above superclass constructor call. We should have probably done this in other PR too

Reply:
> Please add comment like // map from local resources to dtd ids above superclass constructor call.

Done

> We should have probably done this in other PR too.

I update that too